mlir/utils/jenkins/Jenkinsfile

```diff
@@ -63,7 +63,8 @@ void showEnv() {
 }
 
 String dockerArgs() {
-    return "--device=/dev/kfd --device=/dev/dri --group-add video --group-add render -v /etc/passwd:/etc/passwd:ro -v /etc/group:/etc/group:ro"
+    def renderGID = sh(script: "getent group render | cut -d: -f3", returnStdout: true).trim()
+    return "--device=/dev/kfd --device=/dev/dri --group-add video --group-add ${renderGID} -v /etc/passwd:/etc/passwd:ro -v /etc/group:/etc/group:ro"
```

Comment:
You won't be able to directly run script for docker args. 

To be able to exectue script you need to be inside Node/Agent. 

You can pull in #1835  if you want to try scripting for docker args. 

But i would rather recommend just looking at GPU Target first